src/directory/file_watcher.rs

```diff
@@ -104,35 +108,37 @@ mod tests {
         let (tx, rx) = crossbeam::channel::unbounded();
         let timeout = Duration::from_millis(100);
 
-        {
```

Comment:
This is no longer testing that the background thread stops when the watcher is dropped.

Reply:
very good point